easybuild/easyconfigs/p/Perl/Perl-5.24.1-gimkl-2017a.eb

```diff
@@ -0,0 +1,1195 @@
+name = 'Perl'
+version = '5.24.1'
+use_perl_threads = False
+
+homepage = 'http://www.perl.org/'
+description = """Larry Wall's Practical Extraction and Report Language"""
+
+toolchain = {'name': 'gimkl', 'version': '2017a'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+dependencies = [
+    ('DB', '6.2.23'),       # for DB_File module
+    ('expat', '2.2.0'),     # for XML::Parser I think
+    ('libreadline', '6.3'),
+    ('libxml2', '2.9.4'),   # for XML::LibXML
+    ('SQLite', '3.16.2'),   # for DBD::SQLite module
+    ('zlib', '1.2.11'),
+]
+
+builddependencies = [('SAMtools', '0.1.19')]  # NOT A LATER VERSION.  For Bio::DB::Sam to static link to.
+
+source_urls = ['http://www.cpan.org/src/5.0']
+sources = [SOURCELOWER_TAR_GZ]
+
+# We must compile in serial so that Convert::Binary::C is installed
+# in a sane manner.
+maxparallel = 1
+
+#runtest = 'test'  # Several modules have extra dependencies for testing
+
+# Extension versions
+acme_damn_version = '0.08'
```

Comment:
Please include the version in `exts_list` once, and use the `%(version)s` template for the source spec for each extension (cfr. https://github.com/valtandor/easybuild-easyconfigs/pull/16)

This is a nightmare to maintain otherwise imho...